spark/v3.5/spark-runtime/src/integration/java/org/apache/iceberg/spark/SmokeTest.java

```diff
@@ -19,85 +19,84 @@
 package org.apache.iceberg.spark;
 
 import java.io.IOException;
-import java.util.Map;
+import java.nio.file.Files;
+import org.apache.iceberg.ParameterizedTestExtension;
 import org.apache.iceberg.Table;
 import org.apache.iceberg.catalog.TableIdentifier;
-import org.apache.iceberg.spark.extensions.SparkExtensionsTestBase;
-import org.apache.spark.sql.catalyst.analysis.NoSuchTableException;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
-
-public class SmokeTest extends SparkExtensionsTestBase {
-
-  public SmokeTest(String catalogName, String implementation, Map<String, String> config) {
-    super(catalogName, implementation, config);
-  }
-
-  @Before
+import org.apache.iceberg.spark.extensions.ExtensionsTestBase;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(ParameterizedTestExtension.class)
+public class SmokeTest extends ExtensionsTestBase {
+  @AfterEach
   public void dropTable() {
     sql("DROP TABLE IF EXISTS %s", tableName);
   }
 
   // Run through our Doc's Getting Started Example
   // TODO Update doc example so that it can actually be run, modifications were required for this
   // test suite to run
-  @Test
+  @TestTemplate
   public void testGettingStarted() throws IOException {
     // Creating a table
     sql("CREATE TABLE %s (id bigint, data string) USING iceberg", tableName);
 
     // Writing
     sql("INSERT INTO %s VALUES (1, 'a'), (2, 'b'), (3, 'c')", tableName);
-    Assert.assertEquals(
-        "Should have inserted 3 rows", 3L, scalarSql("SELECT COUNT(*) FROM %s", tableName));
+    Assertions.assertThat(scalarSql("SELECT COUNT(*) FROM %s", tableName))
```

Comment:
why not use the static import here?

Reply:
IThe static import is not recommended from the checkstyle. When it's statically imported, the test fails. Here's one of the examples:

```
 > Task :iceberg-spark:iceberg-spark-extensions-3.4_2.12:compileTestJava
Error: eckstyle] [ERROR] /home/runner/work/iceberg/iceberg/spark/v3.5/spark-runtime/src/integration/java/org/apache/iceberg/spark/SmokeTest.java:21:46: Using a static member import should be avoided - org.assertj.core.api.Assertions.assertThat. [AvoidStaticImport]

```